src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -64,6 +64,7 @@ public static class PSVersionInfo
         private static readonly SemanticVersion s_psV62Version = new SemanticVersion(6, 2, 0, preReleaseLabel: null, buildLabel: null);
         private static readonly SemanticVersion s_psV7Version = new SemanticVersion(7, 0, 0, preReleaseLabel: null, buildLabel: null);
         private static readonly SemanticVersion s_psV71Version = new SemanticVersion(7, 1, 0, preReleaseLabel: null, buildLabel: null);
+        private static readonly SemanticVersion s_psV72Version = new SemanticVersion(7, 2, 0, preReleaseLabel: null, buildLabel: null);
```

Comment:
Nit: I wonder we keep old and unsupported versions here

Reply:
@PaulHigin is the PSCompatibleEdition needed for remoting? 
@PowerShell/powershell-maintainers -- what do you think? Should 6.2.0 be removed?

Reply:
Not that I am aware of.

Reply:
I'd remove 1,2,3 and 6 version, and maybe 4 too.